packages/streamid/src/stream-id.ts

```diff
@@ -92,8 +92,8 @@ const TAG = Symbol.for('@ceramicnetwork/streamid/StreamID')
 export class StreamID implements StreamRef {
   protected readonly _tag = TAG
 
-  readonly #type: number
-  readonly #cid: CID
+  private readonly _type: number
```

Comment:
Unfortunately using proper private variables with the hashsign prefix causes `lodash.cloneDeep` to be unable to properly clone StreamIDs.  This caused all kinds of errors as we're now adding StreamIDs into StreamState and StreamMetadata for the first time, both of which have `cloneDeep()` called on them in several places throughout the codebase.

Reply:
:(